lib/game.rb

```diff
@@ -0,0 +1,44 @@
+require 'stringio'
+
+class Game
+
+    def initialize(board, players, input, output)
+        @board = board
+        @players = players
+        @input = input
+        @output = output
+        @current_player_index = 0
+    end
+
+    def play_game
+        loop do
```

Comment:
It might be helpful to have the game announce the winner, in a follow up branch.